bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-general-settings.vue

```diff
@@ -17,16 +23,33 @@
 
 <script>
 import RuleGeneralSettings from '@/components/rule/rule-general-settings.vue'
+import ModuleDescriptionSuggestions from '../module-description-suggestions'
 
 export default {
-  props: ['rule', 'createMode', 'isScriptRule', 'languages', 'mode'],
+  mixins: [ModuleDescriptionSuggestions],
+  props: ['rule', 'module', 'moduleType', 'moduleTypes', 'createMode', 'isScriptRule', 'languages', 'mode'],
   emits: ['newLanguage'],
   components: {
     RuleGeneralSettings
   },
   computed: {
     editable () {
+      console.log('module')
```

Comment:
I will remove this logging, seems like a leftover from development.
FYI to inspect the value of properties during development, the Vue DevTools browser extension is a great tool: https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui/CONTRIBUTING.md#vue-devtools.

Reply:
LOL thanks! I installed this but haven't taken the time to learn how to use it.